README.md

```diff
@@ -1 +1,28 @@
-# language-switcher
\ No newline at end of file
+# Language Switcher
```

Comment:
1. In markdown avoid `<br>`. Use returns.
1. You can put two spaces at the end of a line to create a `<br>`, but you'll almost never need to know this.
1. Try to only have one `<h1>`, as the title of a document, then all following headers should be `<h2>`, and subsections within the H2 sections will have `<h3>`s
1. GitHub doesn't care about having empty lines above headers, but many other Markdown tools do. I always put two empty lines above each header (unless it is the first line of the document)
1. GitHub doesn't care about `-` vs `*` for bullet points, but many other markdown tools do. For safety if you always use `*` it will work everywhere.
1. GitHub *usually* does not care about empty lines around bulleted lists or code fences, by many other Markdown tools do, so it is best to add one empty line above/below them (unless nested in a bulleted list.
1. Try to only have one empty line at the end of files, it just reduces line change noise in PRs.

Reply:
I made these suggested changes.